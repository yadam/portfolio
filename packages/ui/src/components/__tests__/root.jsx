import React from 'react';
import renderer from 'react-test-renderer';
import Root from '../root';

jest.mock('../item', () => 'Item');

it('renders correctly', () => {
  const tree = renderer.create(<Root />).toJSON();
  expect(tree).toMatchSnapshot();
});
