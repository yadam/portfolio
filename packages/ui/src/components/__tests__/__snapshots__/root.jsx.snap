// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiGrid-container-1 MuiGrid-spacing-xs-24-25 MuiGrid-justify-xs-space-around-21"
>
  <Item />
</div>
`;
